resources/sdk/clisdkclient/templates/api_json.mustache

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Is it possible to move this template to a common location as well?

Reply:
Could probably achieve this by storing it in `resources/templates` and passing it in as another `-t` flag in [builder.js](https://github.com/purecloudlabs/platform-client-sdk-common/blob/master/modules/builder.js#L370)

Reply:
I was able to move the script to a common folder, but wasn't able to pass multiple -t options.  I found that the first template folder passed in was used, but not both.

Reply:
Would it be possible to store it in `resources/templates` and keep a sym link to it in each SDK template directory? If that doesn't work then we'll have to leave it as is.